### PR TITLE
Update user

### PR DESCRIPTION
**Pull request by LinaGallardo**
In this branch perform the view "update" user with the gem "devise"
update_users

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```
Comment (Jucerweb) [inline app/views/devise/registrations/edit.html.erb]: Edit and create are the same forms? Why don't you use `partials`?

Comment (LinaGallardo) [reply app/views/devise/registrations/edit.html.erb]: Ok, no problem

Comment (LinaGallardo) [reply app/views/devise/registrations/edit.html.erb]: @Jucerweb, My PR it's ready.

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= render "form", name: "Edit", html: {method: :put} %>
```
Comment (Jucerweb) [inline app/views/devise/registrations/edit.html.erb]: space `html: {method: :put} %>` => `html: { method: :put } %>`

**Review by Jucerweb — APPROVED** [review]
After this comment 👍

**Comment by Jucerweb** [comment]
@LinaGallardo Perfect ❤️

**Review by Jucerweb — APPROVED** [review]

**Pull request merged**